examples/cifar10_nasnet.py

```diff
@@ -85,13 +83,21 @@
     # (std, mean, and principal components if ZCA whitening is applied).
     datagen.fit(X_train)
 
+    # wrap the ImageDataGenerator to yield two label batches [y, y] for each input batch X
```

Comment:
can you also add to the comment to explain why duplicate labels are needed? It seems unusual.

Reply:
Added below